core/src/test/scala/integration/kafka/api/AdminClientTest.scala

```diff
@@ -106,7 +106,7 @@ class AdminClientTest extends IntegrationTestHarness with Logging {
 
     val consumerSummaries = client.describeConsumerGroup(groupId)
     assertEquals(1, consumerSummaries.size)
-    assertEquals(Set(tp, tp2), consumerSummaries.head.assignment.toSet)
+    assertEquals(Set(tp, tp2), consumerSummaries.get.head.assignment.toSet)
```

Comment:
@imandhan Did you see the code I proposed above? It's generally bad form to call `Option.get` because it gives an error that's not very helpful.

If your test fails due to a regression that causes `None` to be returned, it will throw a NoSuchElementException. The code I suggested will say something like: "Expected Some(Set(...)), actual None".